escarleth-ledezma/lesson-1/homework-2-car/src/Car.java

```diff
@@ -0,0 +1,44 @@
+public class Car {
+    private double gas;
+    private boolean powerON;
+    private double maxThankCapacity;
+    private double gasConsumption; //   distance/liters
+    private double mileage;
+
+    public Car(double maxThankCapacity, double gasConsumption) {
+        gas = 0;
+        mileage=0;
+        this.maxThankCapacity = maxThankCapacity;
+        this.gasConsumption = gasConsumption;
+    }
+
+    public void powerOn() {
+        powerON = true;
+    }
+
+    public void powerOff() {
+        powerON = false;
+    }
+
+    public void fillGas(double gasToFill) {
+        gas = (gas + gasToFill) <= maxThankCapacity ? gas+gasToFill : maxThankCapacity;
+    }
+
+    public void move(double distance) {
+        mileage = gas * gasConsumption;
```

Comment:
the mileage should be increment based on the distance
`mileage += distance` but if the condition to move the car are true.